File: Lições/pascal/EXERCICIO16.pas
Program EXERCICIO16 ;

// 16) Escreva um algoritmo que receba o nome e a idade de uma pessoa. Exibir o nome da pessoa
// e a expressão “Maior de Idade” ou a expressão “Menor de Idade”.

Var
nome : string;
idade: integer;

Begin
  Write('Qual é seu nome? ');
  Read(nome);
  Write('Qual é sua idade? ');
  Read(idade);
  If (idade < 18) Then
  Begin
    Write('Olá ',nome,' você é MENOR DE IDADE')
  End
  Else
  Begin
    Write('Olá ',nome,' você é MAIOR DE IDADE');
  End;
End.

File: Lições/python/EXERCICIO16.py
# 16) Escreva um algoritmo que receba o nome e a idade de uma pessoa. Exibir o nome da pessoa
# e a expressão “Maior de Idade” ou a expressão “Menor de Idade”.
nome = str(input('Qual é seu nome? '))
idade = int(input('Quantos anos você tem? '))

if idade >= 18:
  print('Olá {} você é Maior de Idade'.format(nome))
else:
  print('Olá {} você é Menor de Idade'.format(nome))